### PR TITLE
Other email views

### PR DESCRIPTION
**Pull request by PatricKangasmaki**

### CODE REVIEW

**Comment by PatricKangasmaki** [comment]
Adds emails:
 - Project invitation
 - Email verification

Follows the pattern created by the pull request: Forgot password email #271 and should be reviewed only after that one has been approved.

**Comment by PatricKangasmaki** [comment]
Removed test routes and rebased on master.

File: frontend/src/components/emails/ProjectInvitationEmail.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        </a>
+        <div>
+          <p>Or copy the link below to your browser:</p>
+          <p>
+            {baseUrl}/join/{token}
+          </p>
+        </div>
+      </div>
+      <div style={{ ...textStyle, marginTop: '40px', marginBottom: '-40px' }}>
+        <i>Don’t want to join the project? You can ignore this message..</i>
```
Comment (juhoeerola) [inline frontend/src/components/emails/ProjectInvitationEmail.tsx:49]:
```suggestion
        <i>Don’t want to join the project? You can ignore this message.</i>
```

**Review by juhoeerola — CHANGES_REQUESTED** [review]

**Comment by PatricKangasmaki** [comment]
> This could be moved to e.g. the EmailTemplate module as a higher order component, as it starts to be duplicated quite a few times.

Great point, added that. I renamed htmlBox and improved its' styling a bit as well. And also merged the other PR here which had task rating notification emails, I felt like it didn't really need its own branch when it is basically doing the same thing as these other email views, just different texts

**Review by juhoeerola — APPROVED** [review]
👍

**Pull request merged**